CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+## Mapbox Navigation SDK 2.2.0-alpha.2 - In Development
+
+For details on how v2 differs from v1 and guidance on migrating from v1 of the Mapbox Navigation SDK for Android to the v2 public preview, see [2.0 Navigation SDK Migration Guide](https://github.com/mapbox/mapbox-navigation-android/wiki/2.0-Navigation-SDK-Migration-Guide).
+
+### Changelog
+#### Bug fixes and improvements
+- Add `:libnavui-lifecycle:` to share a `MapboxNavigation` instance between `:libnavui-dropin:` and `:libnavui-car:`. [#5112](https://github.com/mapbox/mapbox-navigation-android/pull/5112)
+
+### Mapbox dependencies
```

Comment:
Incrementally adding the changelog seems like it'll always have a initial conflict. could add the next-version-beginning changelog when the previous is released

Reply:
We need to enable the union merge strategy for the changelog file fo this workflow to be reasonable. Refs https://github.com/mapbox/navigation-sdks/issues/1246.

Reply:
Yeah.. we haven't implemented the union merge yet so maybe I could just leave this PR out of it.

Or do we have already decided the strategy?

Reply:
i'm removing the changelog file changes and using the current system on the main line. 

when we deploy our new readme strategy this PR will abide by it